src/containers/Header.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link, Router, IndexRoute, Route, browserHistory } from 'react-router';
+
+
+export default class Header extends Component {
+
+  handleLocation() {
+    const geolocation = require('geolocation');
+    const geo = navigator.geolocation;
+    let lat;
+    let lon;
+    navigator.geolocation.getCurrentPosition(function(position) {
```

Comment:
On Line 13, you stored `navigator.geolocation` into the `geo` constant, why not just use that or not store it at all?